entrypoint.py

```diff
@@ -205,7 +205,9 @@ def _localize(path, config_dir):
     return (config_dir / path).absolute().as_posix()
 
 
-def run_sast(tool, command, env, config_dir, log_file=stdout, run_all=True):
+def run_sast(
+    tool, command, env, config_dir, log_file=stdout, run_all=True, maven_logs=False
```

Comment:
passing tool-specific variables in this function leads to an unmaintainable script with tens of tool-specific variables.

This solution would be ok for a maven-specific tool